api/packages.config

```diff
@@ -1,7 +1,9 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="Microsoft.Bcl" version="1.1.10" targetFramework="portable-net45+win+wp80+MonoTouch10+MonoAndroid10+xamarinmac20+xamarinios10" />
+  <package id="Microsoft.Bcl.Async" version="1.0.168" targetFramework="portable-net45+wp80+win+MonoAndroid10+xamarinios10+MonoTouch10" />
   <package id="Microsoft.Bcl.Build" version="1.0.21" targetFramework="portable-net45+win+wp80+MonoTouch10+MonoAndroid10+xamarinmac20+xamarinios10" />
   <package id="Microsoft.Net.Http" version="2.2.29" targetFramework="portable-net45+win+wp80+MonoTouch10+MonoAndroid10+xamarinmac20+xamarinios10" />
   <package id="Newtonsoft.Json" version="6.0.8" targetFramework="portable-net45+win+wp80+MonoTouch10+MonoAndroid10+xamarinmac20+xamarinios10" />
+  <package id="PortableRest.Signed" version="3.0.1" targetFramework="portable-net45+wp80+win+MonoAndroid10+xamarinios10+MonoTouch10" />

```

Comment:
We shouldn't be including PortableRest or RestSharp, since they break the MobileApp badly.